1.8/administration/installing/custom/advanced.md

```diff
@@ -82,10 +82,11 @@ The DC/OS installation creates these folders:
     - 8.8.4.4
     - 8.8.8.8
```

Comment:
@vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.